src/nodedef.cpp

```diff
@@ -647,6 +671,8 @@ void ContentFeatures::deSerialize(std::istream &is, u16 protocol_version)
 		tmp = readU8(is);
 		if (is.eof())
 			throw SerializationError("");
+		if (tmp > 3)
+			throw SerializationError("unsupported AlphaMode");
```

Comment:
fyi this doesn't work here, the try-catch around it has a no-op handler

Reply:
fixed